iosHyperskillApp/iosHyperskillApp/Sources/Modules/ProgressScreen/Views/Track/ProgressScreenTrackProgressContentView.swift

```diff
@@ -1,3 +1,4 @@
+import shared
```

Comment:
This view shouldn't has a `shared` dependency, provide all necessary data in `ProgressScreenTrackProgressView`

Reply:
@ivan-magda I removed dependency, but view still contains `ProgressScreenViewStateTrackProgressViewStateContentAppliedTopicsStateKs` - I think it's ok because for checking necessity of rendering applied topics progress we anyway need some analogue of sealed interface on platform side and it will be duplicate of view state structure that passed in that view